downstream/modules/platform/proc-aap-migration.adoc

```diff
@@ -0,0 +1,7 @@
+[id="aap-migration_{context}"]
+
+= Migrating data to the {PlatformNameShort} Operator
+
+[role=_abstract]
+
+Once you have set your secret key, postgresql credentials, verified network connectivity and installed the {OperatorPlatform}, you can migrate your data by creating a custom resource controller object.
```

Comment:
Suggestion: Change to imperative voice to fit procedure module (task-oriented) like the following:
 ```suggestion
After you have set your secret key, postgresql credentials, verified network connectivity and installed the {OperatorPlatform}, migrate your data by creating a custom resource controller object.
```

Reply:
Good call! Fixed.